pkg/pillar/cmd/zedrouter/server.go

```diff
@@ -443,12 +443,26 @@ func (hdl networkHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if anStatus != nil {
 		hostname = anStatus.UUIDandVersion.UUID.String()
 	}
+
+	enInfoObj, err := hdl.ctx.subEdgeNodeInfo.Get("global")
+	if err != nil {
+		http.Error(w, http.StatusText(http.StatusNoContent), http.StatusNoContent)
+		return
```

Comment:
Shouldn't this be robust and publish the information we have?

Reply:
Will fix that.